src/sites_toolbox.py

```diff
@@ -94,10 +94,9 @@ def add_all_sites_to_ocf_group(session, site_group_name="ocf"):
     ocf_site_group = get_site_group_by_name(session=session, site_group_name=site_group_name)
     
     site_uuids = [site.site_uuid for site in ocf_site_group.sites]
-
-    sites_added = []
     
     for site in all_sites:
+        sites_added = []
```

Comment:
I'm wondering how best to write a test for a function with a conditional statement. I couldn't quite figure out where to declare `sites_added` and where to put the `return` statement so that `sites_added` would only be returned if it wasn't empty. 

I might have another look at it this morning. :)

I'll have another look this morning.